src/HomePage/AboutMe/AboutMe.scss

```diff
@@ -0,0 +1,26 @@
+.about-section {
+  display: flex;
+  flex-wrap: wrap;
+  align-content: center;
+  min-height: 100vh;
+  background-color: rgb(158, 158, 168);
+
+  &__header {
+    font-family: "Questrial";
```

Comment:
So it seems like you're mostly using this font and the Lato font on the whole site correct? If so then i'd make one of these the default font on the whole site to avoid duplication